pkg/controller/certificates/issuing/issuing_controller_test.go

```diff
@@ -34,6 +34,7 @@ import (
 	internaltest "github.com/jetstack/cert-manager/pkg/controller/certificates/internal/test"
 	testpkg "github.com/jetstack/cert-manager/pkg/controller/test"
 	"github.com/jetstack/cert-manager/test/unit/gen"
+	"github.com/stretchr/testify/require"
```

Comment:
nitpick: I don't think this matches our usual [import ordering](https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports) style.

Reply:
Fixed in 26b074241a0b9a4d752b619918018c760810c3a8. Thanks!